3ev/dev-test
# Popover state

This is a PR for keeping the state of a current popover. I've amended popover.js to include a variable to store the value of the currently open popover. When a new popover is clicked the state variable is updated. If the trigger for the popover is clicked again, the state is cleared.

---

***Testing***

**Test Cases:**

1.  Click episode image once, popover should appear. Click same episode image- popover should disappear.
2.  Click episode image once, popover should appear. Click another episode image, current popover should be removed for previous image and current clicked episode image should appear.
3.  Click episode image once, popover should appear. Click another episode image, current popover should be removed for previous image and current clicked episode image should appear. Click the the same episode image for a second time, popover for episode should appear.

---

***Deployment steps***

Merge branch `popover-state` into `master`

Compile assets: `./bin/node_modules/gulp`
